devdox/app/services/repository_service.py

```
@@ -99,3 +100,66 @@ async def get_all_provider_repos(
         transformed_response = [response_mapper(r) for r in fetched_data.get("data", [])]
         
         return fetched_data["data_count"], transformed_response
+
+
+class RepoManipulationService:
+    def __init__(
+        self,
+        label_store: TortoiseGitLabelStore = Depends(),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)